examples/resources/castai_service_account/resource.tf

```diff
@@ -0,0 +1,5 @@
+resource "castai_service_account" "service_account" {
+  organization_id = organization.id
+  name            = "service-account-name"
+  description     = "service account description"
+}
```

Comment:
What about adding an email and author to the examples?

Reply:
I am not sure as those are read-only fields, adding them would indicate to users that they can set them

Reply:
Oh, I thought we can set the service account email. so that is ok